src/modules/menu/components/search/SearchBar.vue

```diff
@@ -91,3 +91,17 @@ export default {
     },
 }
 </script>
+
+<style lang="scss" scoped>
+@import 'src/scss/variables';
+
+.search-bar {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+
+
+
+</style>
```

Comment:
I'm not sure this is necessary, as the search bar is already centered by its parent (by HeaderWithSearchBar)